docs/gallery-submission.md

```diff
@@ -131,20 +134,11 @@ Installation instructions have two parts:
-- For Tableau Prep Builder:  
-  - Windows: C:\Users\<Windows User>\Documents\My Tableau Prep Repository\Connectors  
-  - MacOS: /Users/<user>/Documents/My Tableau Prep Repository/Connectors  
-- For Tableau Server (Flow web authoring): <Tableau_Server_Installation_Directory>/data/tabsvc/flowqueryservice/Connectors  
-- For Tableau Server (Tableau Prep Conductor): <Tableau_Server_Installation_Directory>/data/tabsvc/flowprocessor/Connectors  
-
-
-To make things easy, we've created a template for your custom installation instructions (included below, markdown friendly). Any additional installation instructions should be appended below the Tableau installation instructions.
-
-| **JDBC** | **ODBC** |
-| ----------- | ----------- |
-| <pre>1. Download the Connector file (.taco).<br /> 2. Move the .taco file here:<br />   - Windows: C:\Users\\[Windows User]\Documents\My Tableau Repository\Connectors<br />   - macOS: /Users/[user]/Documents/My Tableau Repository/Connectors<br />3. Start Tableau and under \*\*Connect\*\*, select the [Connector Name] connector. (\*\*Note:\*\* You'll be prompted if the driver is not yet installed. <br />4. [Connector Name] Driver Installation:<br />   A. Go to the [Driver Download](https://www.driverdownloadlinkhere.com) page.<br />   B. Download the [Connector Name] Driver .jar file and move into the following directory:<br />     - Windows: C:\Program Files\Tableau\Drivers <br />     - macOS: /Users/[user]/Library/Tableau/Drivers<br />5. Relaunch Tableau and connect using the [Connector Name] connector. </pre> | <pre>1. Download the Connector file (.taco). <br />2. Move the .taco file here:<br />   - Windows: C:\Users\\[Windows User]\Documents\My Tableau Repository\Connectors <br />3. Start Tableau and under \*\*Connect\*\*, select the [Connector Name] connector. (\*\*Note:\*\* You'll be prompted if the driver is not yet installed.) <br />4. [Connector Name] Driver Installation: <br />   A. Go to the [Driver Download](https://www.driverdownloadlinkhere.com) page.<br />   B. Download the [Connector Name] Driver and install following the instructions in the readme provided with the client installation. Ensure the 64-bit client version is installed. <br />5. Relaunch Tableau and connect using the [Connector Name] connector. </pre> |
-
+To make things easy, we created templates for your custom installation instructions. These templates, linked below, are markdown friendly. If you have additional installation instructions, append them below the Tableau installation instructions.
+* [Tableau Desktop JDBC Template](./templates/tableau-desktop-jdbc-template.md)
+* [Tableau Prep Builder JDBC Template](./templates/tableau-prep-jdbc-template.md)
+* [Tableau Server (Flow web authoring) Template](./templates/tableau-server-flow-template.md)
+* [Tableau Server (Tableau Prep Conductor) Template](./templates/tableau-server-prep-template.md)
```

Comment:
Couldn't all the Server instructions be collapsed into one? They all duplicate the driver download instructions, which goes to the same directory, it's the taco file that needs to be put into multiple places. I don't think anyone will want to just install the connector to certain services, they want to install in onto the entire server.

Also, we seem to be missing dropping the taco file in the vizql folder, as described [here](https://tableau.github.io/connector-plugin-sdk/docs/run-taco#run-a-packaged-connector-in-tableau-server). So following these instructions someone will enable the connector for Prep on server, but I don't think they'll enable it for the rest of Server.

Reply:
Possibly. Now that we no longer have the ODBC templates, it might be easier to include the other four in the original file. I'll tinker around with that and do another commit.

Reply:
I couldn't figure out an elegant way to display the four templates in the single topic so that folks could copy-and-paste. I left them as distinct files.